src/frontend/grammar.ne

```diff
@@ -25,7 +25,8 @@ import {
 main -> (sexpr_stmt | variants):+ {% d => refSource(program(blockOrSingle(d[0].map(id))), d[0][0]) %}
 
 variant ->
-  (sexpr_stmt | variants):* {% d => refSource(blockOrSingle(d[0].map(id)), d[0].length > 0 ? d[0][0] : undefined) %}
```

Comment:
I couldn't remember why the condition `d[0].length > 0` was there so I removed it.